real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/RobotLogger.java

```diff
@@ -28,12 +28,14 @@
 	public static RobotLogger getInstance() {
 		if (instance == null) {
 			logDir = new File(config.LOG_FILE_LOCATION);
-
-			if (!logDir.exists()) {
-				logDir.mkdirs();
-				System.out.println("Created directory: "
-						+ logDir.getAbsolutePath());
-			}
+			
+			logDir.mkdirs();
+			
+//			if (!logDir.exists()) {
+//				logDir.mkdirs();
+//				System.out.println("Created directory: "
+//						+ logDir.getAbsolutePath());
+//			}
```

Comment:
Note that everyone's computer will only have the dir if there is a file placed in this directory. Git has no concept of an empty directory.